nuxeo-coldstorage/src/main/java/org/nuxeo/coldstorage/ColdStorageConstants.java

```diff
@@ -43,8 +43,12 @@
 
     public static final String COLD_STORAGE_CONTENT_TO_RESTORE_EVENT_NAME = "coldStorageContentToRestore";
 
+    public static final String COLD_STORAGE_CONTENT_TO_RETRIEVE_EVENT_NAME = "coldStorageContentToRetrieve";
```

Comment:
Add `@since`? And below

Reply:
Probably not needed as the first release is still not done :)